### PR TITLE
Created history file with tree and commands

### PR DESCRIPTION
**Pull request by soumyaray**
Folks, I just added a history file — whaaddya think?

### CODE REVIEW

**Comment by rolisanchez** [comment]
Looks good!

**Comment by marandy0905** [comment]
很棒！我覺得超讚！

**Comment by soumyaray** [comment]
👍

**Comment by marandy0905** [comment]
`Something is wrong here!!!!`

File: history.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# History of this project
```
Comment (marandy0905) [inline history.txt:1]: Very helpful title!

**Pull request merged**